### DISCUSSION TITLE
Cant create slash

### DISCUSSION THREAD

**Discussion by Spwn07** [aoijs/aoi.js#430] [Q&A] [answered]
Invalid Form Body type[NUMBER_TYPE_COERCE]: Value "CHAT_INPUT" is not int.

**Comment by Slyrith** [comment, marked as answer]
Interaction moved to 6.1 to make v6 core stable. slash, menu, button, application won’t work. Wait till v6.1 released

**Reply by Spwn07** [reply]
the reply was fast... well, alright then. il wait till v6.1 came out. probably fixed in the next version anyway, thanks

**Reply by Neodevils** [reply]
It's now fixed
